ogre2/src/Ogre2Heightmap.cc

```diff
@@ -158,17 +158,35 @@ void Ogre2Heightmap::Init()
   // Obtain min and max elevation and bring everything to range [0; 1]
   // Terra should support non-normalized ranges but there are a couple
   // bugs preventing that, so it's just easier to normalize the data
-  float minElevation = 0.0;
-  float maxElevation = 0.0;
+  double minElevation = this->descriptor.Data()->MinElevation();
```

Comment:
I think the `MinElevation()` function needs to be backported to ign-common4

Reply:
Ahh I removed that change for now. We can't backport that because it's a virtual function. bb1504170891aea74a4877cd78b93d8e54516891